stand/lua/gfx.lua.8

```diff
@@ -3,12 +3,12 @@
 .\"
 .\" SPDX-License-Identifier: BSD-2-Clause
 .\"
-.Dd February 6, 2024
+.Dd March 29, 2025
 .Dt GFX.LUA 8
 .Os
 .Sh NAME
 .Nm gfx.lua
-.Nd Fx Lua gfx module
+.Nd bootloader graphics module
```

Comment:
"lua gfx module" means that it's documentation for that module named gfx (not graphics).
So this is I think not so good.

Reply:
That's in the name section sir. It renders:
```
gfx.lua(8) - bootloader graphics module
```